### DISCUSSION TITLE
Disappearing Farmers and Their Motives

### DISCUSSION THREAD

**Discussion by dorkmo** [Chia-Network/chia-blockchain#489] [General]
I was pretty amazed by the size of farming plots that came online then vanished after the most recent contest. Are we assuming that the majority of farmers are plotting offline before mainnet? If there are a handful of farmers with massive plots, should we be worried about a 51% attack on day one of mainnet?

![image](https://user-images.githubusercontent.com/1923070/97017984-32c17300-1514-11eb-8678-d48a6d69450c.png)

Would it make sense to have mini-prizes every week to keep more people online?

**Comment by hoffmang9** [comment]
When mainnet starts it will be farming rewards only for 4-6 weeks. Transactions will be locked until a specific block height in the future. As such, there will be plenty to motivate all the potential space to get online and for the work difficulty factors to settle in far before a first transaction. There is some remote risk of a long range 41% attack but the risk of running such would undermine the coins earned by it. Once things settle it becomes harder also.

**Comment by dorkmo** [comment]
Also interesting to note the tick down in chia explorer's "decentralization" chart! 

![image](https://user-images.githubusercontent.com/1923070/97516114-6d863980-1960-11eb-90cf-2caba815fff2.png)

**Comment by BrandtH22** [comment]
These comments and links are outdated. For the most recent information on chain decentralization please refer to the chia dashboard:
https://dashboard.chia.net/d/6S16D9AVk/nakamoto-coefficient?orgId=1

Feel free to open a new discussion if any questions arise.
